udata/core/topic/models.py

```diff
@@ -27,6 +28,18 @@ class Topic(db.Document, db.Owned):
     private = db.BooleanField()
     extras = db.ExtrasField()
 
+    created_at = DateTimeField(default=datetime.utcnow, required=True)
```

Comment:
What about existing ones with `required=True`? Should we fill these somehow?

Reply:
`name` is filled by the user and `slug` is filled automaticaly as is `created_at`

Reply:
I meant existing Topics without a `created_at`

Reply:
We can make a ligration but it won't be really relevant. Or remove the `required`

Reply:
I would say that it is nice to keep the `required`. I've tested your branch and it does fill the created_at value with default value, thus current utcnow.